### PR TITLE
Change default issue template to use different categories

### PR DESCRIPTION
**Pull request by SteveL-MSFT**
## PR Summary

Leverage GitHub ability to have multiple options when user clicks "New Issue" to have better issues opened and direct users to Docs repo and UserVoice for Windows PowerShell issues.

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] User-facing [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed - Issue link:
- **Testing - New and feature**
    - [x] Not Applicable or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
        - [ ] [Add `[feature]` if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)

### CODE REVIEW

**Review by TravisEz13 — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/Bug_Report.md [hunk 1]
```diff
@@ -1,46 +1,45 @@
...
 For Windows PowerShell 5.1 issues, suggestions, or feature requests please use the following link instead:
-- Windows PowerShell [UserVoice](https://windowsserver.uservoice.com/forums/301869-powershell)
+Windows PowerShell [UserVoice](https://windowsserver.uservoice.com/forums/301869-powershell)
+
+This repository is **ONLY** for PowerShell Core 6 issues.
 
-If it is a bug report:
-- make sure you are able to repro it on the latest released version. 
+- Make sure you are able to repro it on the latest released version.
 You can install the latest version from https://github.com/PowerShell/PowerShell/releases
```
Comment (anmenaga) [inline .github/ISSUE_TEMPLATE/Bug_Report.md]: Lines 13-14 are about the same thing.
Probably make 'latest released version' in line 13 a hyperlink with url from line 14; and remove line 14.

File: .github/ISSUE_TEMPLATE/Documentation_Issue.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+name: Documentation Issue 📚
+about: Issues regarding documentation within this repository.
+
+---
+
+# Documentation Issue
+
+Please open documentation issues that is not specifically for documentation within the
```
Comment (anmenaga) [inline .github/ISSUE_TEMPLATE/Documentation_Issue.md]: `is` -> `are`

File: .github/ISSUE_TEMPLATE/Support_Question.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+---
+name: Support Question ❓
+about: If you have a question, you can try asking in the PowerShell Slack channel first.
```
Comment (joeyaiello) [inline .github/ISSUE_TEMPLATE/Support_Question.md]:
```suggestion
about: If you have a question, you can try asking in the PowerShell Slack channel first. If you need official support, refer to the [PowerShell Support Lifecycle](http://aka.ms/pslifecycle)
```

File: .github/ISSUE_TEMPLATE/Support_Question.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+
+---
+
+# Community Resources
```
Comment (joeyaiello) [inline .github/ISSUE_TEMPLATE/Support_Question.md]:
```suggestion
# Support Question

## Official support

[PowerShell Support Lifecycle](http://aka.ms/pslifecycle)

## Community Resources
```

File: .github/ISSUE_TEMPLATE/Windows_PowerShell.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
+---
+name: Windows PowerShell
+about: Windows PowerShell issues/suggestions need to be reported to UserVoice
```
Comment (joeyaiello) [inline .github/ISSUE_TEMPLATE/Windows_PowerShell.md]:
```suggestion
about: Windows PowerShell issues/suggestions need to be reported to [UserVoice](https://windowsserver.uservoice.com/forums/301869-powershell)
```

**Review by joeyaiello — CHANGES_REQUESTED** [review]
Changes are single comments

**Review by joeyaiello — APPROVED** [review]

**Comment by SteveL-MSFT** [comment]
@TravisEz13 this is ready to be merged

**Review by TravisEz13 — APPROVED** [review]

**Pull request merged**